tests/system/tests/bash/wait_creation_cfg.bash

```diff
@@ -1,21 +1,25 @@
 #!/bin/bash
 
 # It waits until either the requested FogLAMP configuration is created or it reaches the timeout.
+count=1
 while [ true ]
 do
     curl -s -X GET http://${FOGLAMP_SERVER}:${FOGLAMP_PORT}/foglamp/category/${1}  | jq '.value'  > /dev/null 2>&1
     result=$?
 
     if [[ "$result" == "0" ]]
```

Comment:
Even in case of host/port unreachable (e.g: curl: (7) Failed to connect to X.X.X.X port YYYY: No route to host) value of $? is 0.
In this case the test will not fail but it should.

Reply:
code improved.